src/ontology/external/nord-qc-failures.md

```diff
@@ -0,0 +1,14 @@
+| MONDO:0001280       |              |                     | Posterior Uveitis          |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasExactSynonym) |
+| MONDO:0002119       |              |                     | Juvenile Ossifying Fibroma |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasExactSynonym) |
+| MONDO:0005731       |              |                     | Acanthocheilonemiasis      |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasExactSynonym) |
+| MONDO:0005761       |              |                     | Elephantiasis              |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasBroadSynonym) |
+| MONDO:0007030       |              |                     | Aarskog Syndrome           |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasExactSynonym) |
+| MONDO:0007306       |              |                     | Klippel-Feil Syndrome      |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasBroadSynonym) |
+| MONDO:0007389       |              |                     | Spondylothoracic Dysplasia |                         |                                                 |                                                |                 |                  | nord     | duplicate_scoped_synonym (oboInOwl:hasExactSynonym) |
+| MONDO:0005676       |              |                     |                            |                         |                                                 | http://purl.obolibrary.org/obo/mondo#nord_rare |                 |                  | nord     | qc-animal-disease-rare (oboInOwl:inSubset)          |
+| MONDO:MONDO:0859214 |              |                     |                            |                         | http://purl.obolibrary.org/obo/mondo#NORD_LABEL | http://purl.obolibrary.org/obo/mondo#nord_rare |                 | MONDO:NORD       | nord     | MONDO:MONDO_pattern (IRI)                           |
```

Comment:
FYI @twhetzel this is where the Banana was identified and will be reported to our stakeholders.

Reply:
This is a consequence of fixing the issue that causes the build file

https://github.com/monarch-initiative/mondo-ingest/pull/628#issuecomment-2481050169